### PR TITLE
Create contrib/grafana.json

### LINKED ISSUE #1119: Add the matrix.org grafana dash

### PR DESCRIPTION
**Pull request by Half-Shot**
Fixes #1119 

Add grafana dashboard for the IRC bridge

### CODE REVIEW

**Review by jaller94 — APPROVED** [review]
Honestly, I did not check this in detail, but it looks like a valid Grafana view config.

**Comment by Half-Shot** [comment]
@jaller94 added a README to give an explanation on how to handle the `bridge` label. Other than that, I think this dash is good to go.

**Review by jaller94 — APPROVED** [review]
Yay for some documentation.

File: contrib/GRAFANA.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Grafana README
+
+You can use the provided `grafana.json` dashboard to view metrics for the bridge on Grafana. The dashboard uses the `bridge` label to name bridges
+in a human friendly manner. The prometheus scrape config section below shows how you can add the bridge label to metrics.
```
Comment (jaller94) [inline contrib/GRAFANA.md:4]: Compound adjective in front of its noun.
```suggestion
in a human-friendly manner. The prometheus scrape config section below shows how you can add the bridge label to metrics.
```

Comment (Half-Shot) [reply contrib/GRAFANA.md:4]: ah

**Pull request merged**